atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -223,6 +169,71 @@ def _send_mail(self, receivers_list, subject, body):
             if s is not None:
                 s.quit()
 
+    def _get_email_from_koji_obj(self, obj):
+        if obj.get('krb_principal') and obj['krb_principal'] != '':
+            return obj['krb_principal'].lower()
+        else:
+            if not self.email_domain:
+                raise RuntimeError("Empty email_domain specified")
+            return '@'.join([obj['owner_name'], self.email_domain])
+
+    def _get_koji_submitter(self, session, metadata):
+        koji_task_id = metadata.get('labels', {}).get('koji-task-id')
```

Comment:
`.get` would not throw exceptions if labels don't have `koji-task-id` key. But since the exceptions are trapped anyway I think this can be removed